setup.cfg

```diff
@@ -65,6 +65,7 @@ tests =
   pytest-mpl
   pytest-regtest
   yamllint
+  py
```

Comment:
Overly pedantic but the list is alphabetical I think to aid reading (well it helps me at least!), so I'd have slotted this in between `pre-commit` and `pytest`.